make/photon/prepare/main.py

```diff
@@ -35,10 +35,12 @@ def main(conf, with_notary, with_clair, with_chartmuseum):
     try:
         validate(config_dict, notary_mode=with_notary)
     except Exception as e:
-        logging.info('Error happend in config validation...')
+        logging.info('Error happened in config validation...')
         logging.error(e)
         sys.exit(-1)
 
+    gen_registry_credentials(config_dict)
```

Comment:
Why not put this to `configs.py` file. Currently, all logics related to parsing  and changing the config_dict should be placed in `parse_yaml_config` function

Reply:
Sure, I've made change to move it to `parse_yaml_config` in a follow-up commit.